cfgov/v1/forms.py

```diff
@@ -141,13 +139,13 @@ def __init__(self, *args, **kwargs):
         self.all_filterable_results = self.get_all_filterable_results()
         page_ids = self.get_all_page_ids()
         self.set_topics(page_ids)
-        self.set_authors(page_ids)
+        self.set_languages(page_ids)
```

Comment:
It's actually unnecessary to have a separate method here for this, nor one that takes `page_ids` since `languages` isn't dependent on them. You could replace this will the line

```python
self.fields['language'].choices = ref.supported_languages
```

Below.

Reply:
Removed in https://github.com/cfpb/consumerfinance.gov/pull/6506/commits/1d5aec108fc0b10390d89419e9d90038e04e3ea9